WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.m

```diff
@@ -416,6 +425,7 @@ - (void)syncHelper:(WPContentSyncHelper *)syncHelper syncContentWithUserInteract
 
 - (void)syncHelper:(WPContentSyncHelper *)syncHelper syncMoreWithSuccess:(void (^)(BOOL))success failure:(void (^)(NSError *))failure
 {
+    [WPAnalytics track:WPAnalyticsStatPostListLoadedMore withProperties:[self propertiesForAnalytics]];
```

Comment:
Subtle thing we can address later - but this will sometimes fire off on initial load instead of pulling in page 2.